vendors/espressif/boards/esp32/aws_demos/application_code/main.c

```diff
@@ -112,11 +111,6 @@ int app_main( void )
 
     if( SYSTEM_Init() == pdPASS )
     {
-        /* A simple example to demonstrate key and certificate provisioning in
-         * microcontroller flash using PKCS#11 interface. This should be replaced
-         * by production ready key provisioning mechanism. */
-        vDevModeKeyProvisioning();
```

Comment:
It's worth considering the potential impact of this change on our first-time getting started workflow. Now, developers that do the quick-start from OCW will have to run the demo twice, and they'll have to figure out how to turn off the provisioning demo and turn on the Hello World demo. The latter is not ideal for a first-time user experience. 

An alternative would be for us to make it so that the provisioning demo is independent from, but not mutually exclusive with, the other demos. In that case, you would likely leave the above line of code, but the provisioning logic would only actually run if the #define for that demo is enabled.

That way, in quick-start mode, we could have both demos enabled. But for developers making the 0-1-N transition, the separation between production and steady-state would be clean.

Reply:
I like this, I will see if this needs to be discussed further, if not I will go ahead with this suggestion